recipes/pymatgen/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "pymatgen" %}
+{% set version = "2019.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2872f431cb1e9dd64e52f89ec669050ea7e90d2acdfcad4d1fc25b1e1d6e8b19
+
+build:
+  skip: true  # [win or py<35]
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
```

Comment:
Why not use the `pip` line as in the example?

Reply:
I was following the recommendation in https://github.com/conda-forge/staged-recipes/wiki/Frequently-asked-questions (8)

Reply:
I removed that recommendation. See https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip